.ci/bindist/linux/debian/focal/Dockerfile

```diff
@@ -2,14 +2,43 @@ FROM ubuntu:focal
+RUN apt-get update && apt-get install -y $DEPS_SCRIPTS $DEPS_SYMBIYOSYS
```

Comment:
github doesn't show it in this view, but just above this change are the comments I was referring to

Reply:
> Maybe consider putting the new stuff at the end of the file.

I put this here because the deps after are the transitive ones for clash, which I think are more likely to change. I'll update the comment instead (unless you have any objections)